lib/screens_web/views/v2/audio/blue_bikes_view.ex

```diff
@@ -0,0 +1,70 @@
+defmodule ScreensWeb.V2.Audio.BlueBikesView do
+  use ScreensWeb, :view
+
+  def render("_widget.ssml", %{
+        destination: destination,
+        minutes_range_to_destination: minutes_range_to_destination,
+        stations: stations
+      }) do
+    ~E|
+    <p>Nearby Blue Bikes stations.</p>
+    <p>It's about a <%= minutes_range_to_destination %> minute bike ride to <%= destination %></p>
+    <p><%= Enum.map(stations, &render_station/1) %></p>
+    |
+  end
+
+  defp render_station(%{status: :normal} = station) do
+    ~E|
+    <s>The bike station at <%= station.name %> is a <%= station.walk_distance_minutes %> minute walk from here</s>
+    <s>It currently has <%= station.num_bikes_available %> bikes available, and <%= station.num_docks_available %> docks available</s>
+    |
+  end
+
+  defp render_station(%{status: :valet} = station) do
+    ~E|
+    <s>The bike station at <%= station.name %> is a <%= station.walk_distance_minutes %> minute walk from here</s>
+    <s>It currently has valet service, with unlimited bikes available</s>
+    |
+  end
+
+  defp render_station(%{status: :out_of_service} = station) do
+    ~E|
+    <s>The bike station at <%= station.name %> is currently not in service</s>
+    |
+  end
+end
+
+[
+  %{
+    arrow: :n,
+    id: "273",
+    name: "Forest Hills",
+    num_bikes_available: 7,
+    num_docks_available: 32,
+    status: :normal,
+    walk_distance_minutes: 1
+  },
+  %{
+    arrow: :e,
+    id: "279",
+    name: "Williams St at Washington St",
+    num_bikes_available: 3,
+    num_docks_available: 16,
+    status: :normal,
+    walk_distance_minutes: 10
+  },
+  %{
+    arrow: :ne,
+    id: "11",
+    name: "Longwood Ave at Binney St",
+    status: :valet,
+    walk_distance_minutes: 30
+  },
+  %{
+    arrow: :e,
+    id: "61",
+    name: "Boylston St at Fairfield St",
+    status: :out_of_service,
+    walk_distance_minutes: 45
+  }
+]
```

Comment:
Is this here intentionally for documentation purposes?

Reply:
Whoops! I pasted it in as a sample of the serialized data while working on the view functions and intended to remove it after I was done. Funny that the compiler/Credo/Dialyzer didn't complain about it at all. I guess the compiler just happily evaluates expressions like this, just like everything else, and since they have no side effects (like defining a module or a function), it's not an issue.